src/toast.tsx

```diff
@@ -1,75 +1,110 @@
-import {
-    React, FC, useEffect, useRef, cx, useState,
-} from './common.js'
+import { React, FC, useEffect, useRef, cx, useState } from './common.js'
 import { createRoot } from 'react-dom/client'
+import BSToast from 'bootstrap/js/dist/toast'
```

Comment:
sorry, we can't do a top level import here because the bootstrap toast lib reads `window` and that breaks server-side rendering.     However it is safe to import types at top-level, so you can do:
`import type BSToastT from 'bootstrap/js/dist/toast'`

and then import the actual lib on demand using `import()` as it was before